README.md

```diff
@@ -1,32 +1,354 @@
-# model-spinal-rootlets
+# Spinal Nerve Rootlets Segmentation on T2w Images
 
 ## 1) Project Overview
 
-The goal of this project is to develop a method based on the DL technique to segment and locate the spinal roots.
-The localization of these nerves is especially important for the electrode-based treatment of spinal cord injuries.
+The goal of this project is to develop a deep learning (DL)-based method to segment and locate the spinal nerve rootlets.
 
-## 2) Work done
+A script to convert nerve segmentation to spinal level is also available. 
 
-After [reviewing](https://github.com/ivadomed/model-spinal-rootlets/issues/1#issue-1706345176) the available datasets,
-it was determined that none of them were suitable.
-Therefore, a new dataset was used, and each sample was labeled with a binary value representing the presence of
-spinal rootlets (0: False, 1: True).
+[Google slides](https://docs.google.com/presentation/d/1ZHliup_Mtk0OcmI1qkwmOIY7Ml4mO6vewIwFQjMMMPo/edit?usp=sharing) to
+summarize this project are also available.
 
-The original dataset can be
-found [here](https://openneuro.org/datasets/ds004507/versions/1.0.1). [review](https://github.com/ivadomed/model-spinal-rootlets/issues/1#issue-1706345176)
-An initial labeling of 12 subjects was performed to train one fold of the nnUNetV2 3d_fullres model (refer
-to [issue#5](https://github.com/ivadomed/model-spinal-rootlets/issues/5)).
-The resulting model achieved a Dice score of approximately 0.52.
+## 2) Work Done
 
-After conducting inference on the entire dataset and performing manual corrections, the dataset was expanded to include
-20 subjects. However, two subjects were subsequently excluded due to unsatisfactory label quality (
-see [issue#7](https://github.com/ivadomed/model-spinal-rootlets/issues/7)).
-A full training on 5 folds with 250 epochs resulted to a Dice score of approximately 0.58.
+### A) Literature and Data Review
 
-Inference was further conducted on T2w data from 267 subjects from
-the [spinegeneric](https://github.com/spine-generic/data-multi-subject#spine-generic-public-database-multi-subject)
-dataset (release XXX), which had an isotropic voxel size of 0.8mm. Some of the predictions from this inference were manually
-corrected, and the resulting dataset was merged with the previous dataset, resulting in a final dataset containing 38
-subjects.
+No articles were found on this topic.
 
-## 3) Used tools 
+After reviewing the available
+datasets ([issue#1](https://github.com/ivadomed/model-spinal-rootlets/issues/1#issue-1706345176)), it was determined
+that none of them were suitable.
+
+### B) Dataset Creation
+
+Initially, the plan was to test with a binary label (0: no rootlet, 1: rootlet). In the following text, the datasets
+will be numbered as D1, D2, etc., and the models as M1, M2, etc.
+
+**Active learning pathway:**
+![pipeline](pipeline-graph.png)
```

Comment:
Having "Subjects with uncertain labeling are drop" in a separate box is a little bit confusing.
The "Subjects with uncertain labeling are drop" sentence could be appended to "Manual correction":

"Manual correction (subjects with uncertain labeling were excluded)"